Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM golang:1.20-alpine AS builder
+FROM cgr.dev/chainguard/go:1.20 AS builder
```

Comment:
We will also want to use the `cgr.dev/chainguard/static` image in the `Dockerfile.goreleaser` file. That's the docker image that actually gets released.